examples/ChatQnA/deploy/gaudi.md

```diff
@@ -0,0 +1,974 @@
+# Single node on-prem deployment with vLLM or TGI on Gaudi AI Accelerator
+
+This deployment section covers single-node on-prem deployment of the ChatQnA
+example with OPEA comps to deploy using vLLM or TGI service. There are several
+slice-n-dice ways to enable RAG with vectordb and LLM models, but here we will
+be covering one option of doing it for convenience : we will be showcasing  how
+to build an e2e chatQnA with Redis VectorDB and neural-chat-7b-v3-3 model,
+deployed on IDC. For more information on how to setup IDC instance to proceed,

```

Comment:
I don't think IDC is a well known term.  Should probably spell it out on first use.